test/parallel/test-cluster-server-restart-none.js

```diff
@@ -10,15 +10,19 @@ if (cluster.isMaster) {
   worker1.on('listening', common.mustCall(() => {
     const worker2 = cluster.fork();
     worker2.on('exit', (code, signal) => {
-      assert.strictEqual(code, 0, 'worker2 did not exit normally');
-      assert.strictEqual(signal, null, 'worker2 did not exit normally');
+      assert.strictEqual(code, 0, `worker2 did not exit normally. exited with\
+                         code ${code}`);
+      assert.strictEqual(signal, null, `worker2 did not exit normally. exited\
+                         with signal ${signal}`);
```

Comment:
The basic change is good but we try to avoid using multi-line template literals in core. This can be improved by...

```js
      assert.strictEqual(code, 0, 
                         'worker2 did not exit normally.' +
                         `exited with code ${code}`);
      assert.strictEqual(signal, null,
                         'worker2 did not exit normally. ' +
                         `exited with signal ${signal}`);
```

Reply:
That can be fixed on landing tho :-)